$MirOS: ports/databases/postgresql/patches/patch-config_programs_m4,v 1.1.7.1 2005/03/18 15:44:16 tg Exp $
--- config/programs.m4.orig	Tue May 23 21:28:22 2006
+++ config/programs.m4	Thu Feb 22 20:25:47 2007
@@ -95,7 +95,7 @@ for pgac_rllib in $READLINE_ORDER ; do
       # recognize dependent libraries; assume curses is needed if we didn't
       # find any dependency.
       case $host_os in
-        netbsd* | openbsd* | irix*)
+        mirbsd* | netbsd* | openbsd* | irix*)
           if test x"$pgac_lib" = x"" ; then
             pgac_lib=" -lcurses"
           fi ;;
